src/main/java/seedu/address/logic/parser/ListCommandParser.java

```diff
@@ -0,0 +1,44 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.util.List;
+
+import seedu.address.logic.commands.ListCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new FindCommand object
```

Comment:
I think this should be `... creates a new ListCommand Object`?

Reply:
Oh right you are, let me change it, thanks!